<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winamp_playlist</class>
 <widget class="QMainWindow" name="winamp_playlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WINAMP PLAYLIST</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/winamp-icon.png</normaloff>img/winamp-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(27, 0, 49);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>514</y>
      <width>42</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
border-top: 4px double rgb(253, 253, 253);
border-right: 4px double #DFDBDD;
border-bottom: 4px double #BCB8BA;
border-left: 8px double rgb(42, 42, 42);
color: rgb(6, 6, 6);
</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rem_btn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>514</y>
      <width>42</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
border-top: 4px double rgb(253, 253, 253);
border-right: 4px double #DFDBDD;
border-bottom: 4px double #BCB8BA;
border-left: 8px double rgb(42, 42, 42);
color: rgb(4, 4, 4);
</string>
    </property>
    <property name="text">
     <string>REM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sel_btn_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>514</y>
      <width>42</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
border-top: 4px double rgb(253, 253, 253);
border-right: 4px double #DFDBDD;
border-bottom: 4px double #BCB8BA;
border-left: 8px double rgb(42, 42, 42);
color: rgb(11, 11, 11);
</string>
    </property>
    <property name="text">
     <string>SEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="misc_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>514</y>
      <width>42</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
border-top: 4px double rgb(253, 253, 253);
border-right: 4px double #DFDBDD;
border-bottom: 4px double #BCB8BA;
border-left: 8px double rgb(42, 42, 42);
color: rgb(9, 9, 9);
</string>
    </property>
    <property name="text">
     <string>MISK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="opt_btn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>520</y>
      <width>42</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
border-top: 4px double rgb(253, 253, 253);
border-right: 4px double #DFDBDD;
border-bottom: 4px double #BCB8BA;
border-left: 8px double rgb(42, 42, 42);
color: rgb(9, 9, 9);
</string>
    </property>
    <property name="text">
     <string>LIST
OPTS</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="title_lcd">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>520</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(170, 255, 0);
selection-color: rgb(168, 252, 0);
border-right: 2px outset #DFDBDD;
border-bottom: 2px outset #BCB8BA;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>true</bool>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="time_song_text">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>550</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);
selection-color: rgb(168, 252, 0);
border-right: 2px outset #DFDBDD;
border-bottom: 2px outset #BCB8BA;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>true</bool>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="title_lcd_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>520</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(170, 255, 0);
selection-color: rgb(168, 252, 0);
border-right: 2px outset #DFDBDD;
border-bottom: 2px outset #BCB8BA;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>false</bool>
    </property>
    <property name="undoRedoEnabled">
     <bool>true</bool>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pl_play_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>550</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(251, 255, 172);</string>
    </property>
    <property name="text">
     <string>▶️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pl_pause_btn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>550</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(251, 255, 172);</string>
    </property>
    <property name="text">
     <string>⏸️</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pl_stop_btn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>550</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(251, 255, 172);</string>
    </property>
    <property name="text">
     <string>⏹ </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pl_next_btn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>550</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(251, 255, 172);</string>
    </property>
    <property name="text">
     <string>⏩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pl_back_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>550</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(251, 255, 172);</string>
    </property>
    <property name="text">
     <string>⏪ </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pl_download_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(251, 255, 172);</string>
    </property>
    <property name="text">
     <string>⏏️</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_songs">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>5</y>
      <width>590</width>
      <height>504</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
